python_practices

```diff
@@ -0,0 +1,5 @@
+numbers = [71,72,73,74,75,76,77,78,79,80]
+squared_numbers = [number ** 2 for number in numbers]
+list1=[squared_numbers]
+numbers.extend(list1)
```

Comment:
We need a seperate list of squares of each number. So why extending the existing numbers list?